dispatch/static/manager/src/js/reducers/EntitiesReducer.js

```diff
@@ -78,6 +78,10 @@ export default function entitiesReducer(state = initialState, action) {
       images: R.merge(state.images, action.payload.entities.images),
       image: R.merge(state.image, action.payload.entities.images)
     })
+  case `${types.CREATE_IMAGE}_FULFILLED`:
+    return R.merge(state, {
+      images: R.merge(state.images, action.payload.entities.images)
```

Comment:
you can also lift the inner merge(s) into `mergeWith`, but maybe overly concise:
```js
return R.mergeWith(R.merge, state, {images: action.payload.entities.images})